modmesh/core.py

```diff
@@ -105,6 +107,34 @@ def _load():
     for name in __all__:
         globals()[name] = getattr(_impl, name)
 
+    # Try to find the thirdparty, if failed to find it
```

Comment:
The whole logic should be placed in a function and documented in the docstring.  It's hard to understand.